networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/peers/Eth2Peer.java

```diff
@@ -97,7 +97,8 @@ public void updateStatus(final PeerStatus status) {
             },
             error -> {
               LOG.error("Failed to validate updated peer status", error);
-              disconnectCleanly(DisconnectReason.UNABLE_TO_VERIFY_NETWORK);
+              disconnectCleanly(DisconnectReason.UNABLE_TO_VERIFY_NETWORK)
+                  .finish(err -> LOG.debug("Error while disconnecting from peer {}", this, err));
```

Comment:
Since `disconnectCleanly` already has an `exceptionally` clause to handle the goodbye not sending, this should always complete normally and we can just use `.reportExceptions()`.  If it does wind up failing, it's not for any expected reason and should be reported very loudly.